packages/app-desktop/gui/JoplinCloudSignUpCallToAction.tsx

```diff
@@ -0,0 +1,22 @@
+import { _ } from '@joplin/lib/locale';
+import * as React from 'react';
+import bridge from '../services/bridge';
+
+const JoplinCloudSignUpCallToAction = () => {
+
+	const onJoplinCloudSingUpClick = async () => {
+		await bridge().openExternal('https://joplinapp.org/plans/');
+	};
+
+	return <div className="joplin-cloud-sign-up">
+		<p>{_('Don\'t have a Joplin Cloud account? ')}
+			<a
+				href="#"
+				onClick={onJoplinCloudSingUpClick}
+			>{_('Sign-up here')}</a>
```

Comment:
Consider changing this to "Sign up" from "Sign-up":
```suggestion
			>{_('Sign up here')}</a>
```

According to a quick web search, "sign-up" and "signup" are generally nouns, while "sign up" is a verb. See:
- https://english.stackexchange.com/a/71931 (and the sources it cites)
- https://writingexplained.org/sign-up-or-sign-up-difference
- https://css-tricks.com/sign-up-vs-signup/

Reply:
@pedr, it doesn't look like this comment was taken into consideration since you still had "sign-up" in the text?